docs/0006-flat-layout.md

```diff
@@ -0,0 +1,68 @@
+# OCFL Community Extension 0006: Flat OCFL Storage Layout
+
+  * **Extension Name:** 0006-flat-layout
+  * **Authors:** Peter Winckles
+  * **Minimum OCFL Version:** 1.0
+  * **Obsoletes:** n/a
+  * **Obsoleted by:** n/a
+
+## Overview
+
+This storage root extension describes a simple flat OCFL storage layout. OCFL object identifiers are mapped directly to directory names that are direct children of the OCFL storage root directory.
+
+The limitations of this layout are filesystem dependent, but are generally as follows:
+
+* The size of object IDs cannot exceed the maximum allowed directory name size (eg. 255 characters)
+* Object IDs cannot include characters that are illegal in directory names
+* Performance may degrade as the size of a repository increases because every object is a direct children of the storage root
```

Comment:
typo/grammar: "...every object is a direct children of..." should be "...every object is a direct child of..."

Reply:
fixed